packages/ui/src/components/va-data-table/hooks/useColumns.ts

```diff
@@ -71,25 +31,22 @@ export const buildTableColumn = (source: TTableColumnSource, initialIndex: numbe
   }
 }
 
-const buildColumnsFromItems = (rawItems: Ref<ITableItem[]>) => {
-  return Object.keys(merge({}, ...rawItems.value)).map(buildTableColumn)
+const buildColumnsFromItems = (items: Ref<ITableItem[]>) => {
+  return Object.keys(merge({}, ...items.value)).map(buildTableColumn)
 }
 
-const buildNormalizedColumns = (rawColumns: Ref<TTableColumnSource[]>) => {
-  return rawColumns.value.map(buildTableColumn)
+const buildNormalizedColumns = (columns: Ref<TTableColumnSource[]>) => {
+  return columns.value.map(buildTableColumn)
```

Comment:
```suggestion
const buildNormalizedColumns = (columns: TTableColumnSource[]) => {
  return columns.map(buildTableColumn)
```

How about remove Ref from these methods. This will remove overwhelming `toRef`.

Reply:
done